src/code/code_800EC960.c

```diff
@@ -3,91 +3,105 @@
 
 typedef struct {
     f32 unk_00;
-    f32 unk_04;
-    s8 unk_08;
-    s8 unk_09;
-    s8 unk_0A;
+    f32 oldFreqScale;
+    s8 oldReverb;
+    s8 oldPanSigned;
+    s8 oldReverbFlg;
     u8 unk_0B;
     u8 unk_0C;
 } unk_s1;
 
 typedef struct {
-    f32 unk_00;
-    f32 unk_04;
-    f32 unk_08;
-    s32 unk_0C;
-} unk_s2;
+    f32 value;
+    f32 target;
+    f32 step;
+    s32 remainingFrames;
+} Lerp;
 
 typedef struct {
     u16 unk_00;
     u16 unk_02;
     u8 unk_04[100];
 } D_801306DC_s; // size = 0x68
 
-void Audio_SetVolScale(u8, u8, u8, u8);
-void func_800F5550(u16);
-void func_800F5E18(u8 arg0, u16 arg1, u8 arg2, s8 arg3, s8 arg4);
-void func_800F4784(unk_s2* arg0);
-void func_800F56A8(void);
-void func_800F6FB4(u8);
-s32 func_800F491C(u8 arg0);
+enum AudioDebugPage {
+    PAGE_NON = 0,
+    PAGE_SOUND_CONTROL = 1,
+    PAGE_SPEC_INFO = 2, // unused
+    PAGE_HEAP_INFO = 3,
+    PAGE_GROUP_TRACK_INFO = 4, // unused
+    PAGE_SUB_TRACK_INFO = 5,
+    PAGE_CHANNEL_INFO = 6, // unused
+    PAGE_INTERFACE_INFO = 7,
+    PAGE_SE_FLAG_SWAP = 8,
+    PAGE_BLOCK_CHANGE_BGM = 9,
+    PAGE_NATURAL_SOUND_CONTROL = 10, // unused
+    PAGE_OCARINA_TEST = 11,
+    PAGE_SE_PARAMETER_CHANGE = 12,
+    PAGE_SCROLL_PRINT = 13,
+    PAGE_FREE_AREA = 14,
+    PAGE_LIM = 15
```

Comment:
Done, though I'm not completely sold on the name "MAX"... it sounds like it equals the max of all real enum variants while in reality it's that + 1.

Reply:
afaik the _MAX convention started because its what they originally did in some cacses
`ASSERT(gMaxActorId == ACTOR_ID_MAX, "MaxProfile == ACTOR_DLF_MAX", "../z_actor.c", UNK_LINE);`

im fine with it personally, but some people have been confused from it in the past. so it may be worth talking about again

Reply:
`COUNT` or something related would be an obvious alternative (as in `ARRAY_COUNT`).